peachjam/models/judgment.py

```diff
@@ -343,12 +340,12 @@ def pre_save(self):
 
 
 class CaseNumber(models.Model):
-    string_override = models.CharField(
-        _("string override"),
+    full_case_number_as_printed_on_judgment = models.CharField(
```

Comment:
As above, the `string_override` must remain, just the label part of the definition `_("full case number...")` must change.